.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Request a new format specification to be created'
+labels: spec-request
+---
+<!--
+A request for a format specification consist of 2 YAML blocks followed by a free-form text description.
```

Comment:
Grammar nitpick: "... consist<ins>s</ins> of 2 YAML blocks ..."

Reply:
Thanks, really my fault.